.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,49 @@
+# More info:
```

Comment:
I suggest naming the file with a proper `.yaml` extension. (Yes, I'm tilting at windmills. :slightly_smiling_face:)